src/ts/localization/i18n.ts

```diff
@@ -0,0 +1,178 @@
+import vocabularyDe from './languages/de.json';
+import vocabularyEn from './languages/en.json';
+import { LocalizationConfig } from '../uimanager.js';
+
+export const defaultTranslations: BitmovinPlayerUiTranslations = {
```

Comment:
If the type id changed to `Vocabularies`, please call this `defaultVocabularies`